### DISCUSSION TITLE
How can I contribute?

### DISCUSSION THREAD

**Discussion by ShivamMenda** [vedantbahel/viteED#22] [Q&A]
Hey @vedantbahel I really like the idea of the app and I'm also kind of a newbie in flutter, I do mostly frontend work and I'm learning backend right now. I was wondering how I could help you guys with this project by being a contributor?

**Comment by vedantbahel** [comment]
Hi Shivam, 
Thank you for showing interest in contributing to viteED. There will soon be certain task listed in the "Issues" tab that we seek contributions for. You may checkout those and see if you can contribute to any. 
Cheers!

**Reply by ShivamMenda** [reply]
@vedantbahel Sure Thanks!
